app/handlers.py

```diff
@@ -117,7 +117,7 @@ def validate_case(case_json):
                 if resp.status == 404:
                     logger.info("Attempt to use an invalid access code", client_ip=client_ip)
                     flash(request, INVALID_CODE_MSG)
-                    raise HTTPFound("/")
+                    raise HTTPAccepted()
```

Comment:
This will return the string `202: Accepted` to the browser. I think you would still need to use the `HTTPFound("/")` to handle redirect here?

Reply:
`return aiohttp_jinja2.render_template("index.html", request, {}, status=202)` should do it.